source/_integrations/ecobee.markdown

```diff
@@ -173,6 +176,10 @@ The `Compressor minimum temperature` number represents the outdoor temperature a
 
 Check your heat pump Owners' Manual prior to adjusting this value; do not adjust it below the rated minimum operating temperature of the heat pump. **Failure to observe the rated minimum operating temperature can cause damage to the system**
 
+The `Auxiliary maximum outdoor temperature` number represents the outdoor temperature at which the auxiliary heat will not run. This is represented in the temperature units you have selected in Home Assistant. This is also represented in the thermostat user interface. When the outdoor temperature is above this value, auxiliary heat will not be used. Be careful with this setting, as it can incur additional utility costs from using a less-efficient heat source.
```

Comment:
```suggestion
The **Auxiliary maximum outdoor temperature** number represents the outdoor temperature at which the auxiliary heat will not run. This is represented in the temperature units you have selected in Home Assistant. This is also represented in the thermostat user interface. When the outdoor temperature is above this value, auxiliary heat will not be used. Be careful with this setting, as it can incur additional utility costs from using a less-efficient heat source.
```